Classes/YLTableView.m

```diff
@@ -71,7 +73,7 @@ - (void)registerClass:(Class)headerFooterViewClass forHeaderFooterViewReuseIdent
   }
 }
 
-- (YLTableViewCell *)sizingCellForReuseIdentifier:(NSString *)reuseIdentifier {
+- (nonnull YLTableViewCell *)sizingCellForReuseIdentifier:(nonnull NSString *)reuseIdentifier {
   NSAssert(reuseIdentifier != nil, @"Must have a reuse identifier.");
```

Comment:
1. Sounds good, where do we do that though?  I searched for NSAssert and see the `x != nil` style used
2. I think I'd want to leave the NSAssert in.  Nullability annotations won't throw exceptions and also don't warn for cases like setting a pointer to nil, then passing that pointer in to a nonnull param.